CRM/Utils/VersionCheck.php

```diff
@@ -336,6 +336,13 @@ private function writeCacheFile($contents) {
     }
   }
 
+  /**
+   * Removes cached version info.
+   */
+  public function flushCache() {
+    unlink($this->cacheFile);
```

Comment:
@colemanw I think this needs a guard -- e.g. `if (file_exists(...)) unlink(...)`